pulpcore/cli/core/user.py

```diff
@@ -1,8 +1,8 @@
 import click
+from pulp_glue.common.context import PluginRequirement  # PulpEntityContext,; pass_entity_context,
```

Comment:
Is there a reason to leave the commented-out imports here?

Reply:
Not at all. TBH they were there before and i used sed to update _all_ the files.